src/DotNet.Status.Web/Controllers/TelemetryController.cs

```diff
@@ -0,0 +1,78 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Kusto;
+using Kusto.Ingest;
+using System.Collections.Generic;
+
+namespace DotNet.Status.Web.Controllers
+{
+    [Route("api/telemetry")]
+    [ApiController]
+    public class TelemetryController : ControllerBase
+    {
+        private readonly ILogger<TelemetryController> _logger;
+        private readonly IOptionsSnapshot<TelemetryOptions> _options;
+        private readonly IKustoIngestClient _client;
+
+        public TelemetryController(
+            ILogger<TelemetryController> logger,
+            IOptionsSnapshot<TelemetryOptions> options,
+            IKustoIngestClient client = null)
+        {
+            _logger = logger;
+            _options = options;
+            _client = client;
+        }
+
+        [HttpPost("collect/ArcadeValidation")]
+        public async Task<IActionResult> CollectArcadeValidation([Required] ArcadeValidationData data)
+        {
+            _logger.LogInformation("Start Collect Arcade Validation data");
+
+            TelemetryOptions options = _options.Value;
+
+            if(null == options)
+            {
+                _logger.LogError("TelemetryOptions were not loaded.");
+                return StatusCode(500);
+            }
+
+            if (null == _client && string.IsNullOrEmpty(options.KustoIngestConnectionString))
```

Comment:
Actually, we should probably just inject a "KustoClientProvider" in Startup, and let this guy not worry about it (it's something elses job to figure out how we talk to Kusto... this class doesn't really care).

Reply:
Is that going to be okay since we create the KustoIngestClient based on the option values in the appsettings.json? (We could have a different Kusto db to interact with in the future)

Reply:
Well, you'll want to set up the DI so that it does that, but yes. :-)

Reply:
Set up KustoIngestExtension class to handle DI for creating the KustoIngestClient.